bitcoinology/what-is-lightning-network.md

```diff
@@ -0,0 +1,90 @@
custodial service by a company called Galloy. And when you scan their QR codes, maybe later one of you might want to try that one particularly, you'll see that you actually get sent to a webpage, not to a Lightning wallet. You have to go to a webpage, and the webpage then hosts the Lightning payment invoice for this person, even though again, it's custodial. They themselves are not holding the keys. So it's a good example of how people make trade-offs in the real world. Is that just for their business when it comes to tax or is that help in accounting? I think it's a complicated question as to why they've gone down that road, but I think it's mostly about the technological difficulty of it makes it a lot simpler for a person who doesn't have a detailed technical background or the time to get involved in nodes and whatnot. I think it's an interesting, I wanna move on, but I think it's a good question. Maybe we can talk about it in the second hour, yeah? Chivo, I wanna mention about Chivo, it's very important. Chivo, this is an example of the better aspect of Chivo, which their ATMs actually successfully, like will give you dollars in return for Bitcoin. But it's main chain only on the ATMs. Now their mobile wallet is quite frankly a complete disaster. I mean, by all accounts and by my own experience, a complete disaster. It does not work for Lightning anyway. It has by default a US dollar interface. By the way, guys, you were there. Correct me if I'm wrong, yeah? It has a US dollar interface by default. You can choose Bitcoin, but that will default to main chain Bitcoin. And then if you look really carefully, it will go to Lightning, yes. I was just gonna, you didn't finish, let me finish. The third option, if you look carefully, there is a button to choose Lightning, right? And perhaps you're gonna correct me in saying this, but from what I heard from other people and from myself, it tended to be the case that even if you could get a Lightning payment to work, that it wouldn't show up on the app. Did you find it differently? Yeah. It works for me all the time. It works for you every time, wow. So, okay. I was thinking about the merchant version. Yes. Personal version. And maybe the merchant one didn't work. The personal one worked. Right, that sounds right. Because I just remembered now, I did pay one person, it was like a barista, a coffee place, I paid his personal Chivo wallet, fine, you're exactly right. But the merchant one, I tried it a couple of times and I heard several other people saying that when you're in a restaurant and you try to pay them, your wallet would show that you'd paid $10 and their wallet would show nothing. So it was a disaster. I had to pay twice. But there have been several other problems in the early days of Chivo, and from what I've heard from people, it's really bad. And I'm kind of upset about it because I think it's kind of damaged the ordinary everyday Salvadorian's perspective of what Bitcoin adoption might be. 
```

Comment:
```suggestion
How can I summarize this quickly? Because I'm kind of running on a little bit too long. So you've got the Bitcoin, it's very much a tale of two cities, the El Salvador experience. You've got the Bitcoin beach, famous project, in a place called El Zonte, literally on the beach, of course. Very small place, very, we could say underdeveloped. Few restaurants, few hotels, and basically everyone there has, like as a project, started using Bitcoin and also Lightning for payments for various things. And they tend to use something called the Bitcoin Beach Wallet, which is a really strange hybrid. It's not really like the things I just showed you, which you just install on your phone. It's a centralized custodial service by a company called Galoy. And when you scan their QR codes, maybe later one of you might want to try that one particularly, you'll see that you actually get sent to a webpage, not to a Lightning wallet. You have to go to a webpage, and the webpage then hosts the Lightning payment invoice for this person, even though again, it's custodial. They themselves are not holding the keys. So it's a good example of how people make trade-offs in the real world. Is that just for their business when it comes to tax or is that help in accounting? I think it's a complicated question as to why they've gone down that road, but I think it's mostly about the technological difficulty of it makes it a lot simpler for a person who doesn't have a detailed technical background or the time to get involved in nodes and whatnot. I think it's an interesting, I wanna move on, but I think it's a good question. Maybe we can talk about it in the second hour, yeah? Chivo, I wanna mention about Chivo, it's very important. Chivo, this is an example of the better aspect of Chivo, which their ATMs actually successfully, like will give you dollars in return for Bitcoin. But it's main chain only on the ATMs. Now their mobile wallet is quite frankly a complete disaster. I mean, by all accounts and by my own experience, a complete disaster. It does not work for Lightning anyway. It has by default a US dollar interface. By the way, guys, you were there. Correct me if I'm wrong, yeah? It has a US dollar interface by default. You can choose Bitcoin, but that will default to main chain Bitcoin. And then if you look really carefully, it will go to Lightning, yes. I was just gonna, you didn't finish, let me finish. The third option, if you look carefully, there is a button to choose Lightning, right? And perhaps you're gonna correct me in saying this, but from what I heard from other people and from myself, it tended to be the case that even if you could get a Lightning payment to work, that it wouldn't show up on the app. Did you find it differently? Yeah. It works for me all the time. It works for you every time, wow. So, okay. I was thinking about the merchant version. Yes. Personal version. And maybe the merchant one didn't work. The personal one worked. Right, that sounds right. Because I just remembered now, I did pay one person, it was like a barista, a coffee place, I paid his personal Chivo wallet, fine, you're exactly right. But the merchant one, I tried it a couple of times and I heard several other people saying that when you're in a restaurant and you try to pay them, your wallet would show that you'd paid $10 and their wallet would show nothing. So it was a disaster. I had to pay twice. But there have been several other problems in the early days of Chivo, and from what I've heard from people, it's really bad. And I'm kind of upset about it because I think it's kind of damaged the ordinary everyday Salvadorian's perspective of what Bitcoin adoption might be. 
```